docs/missioncontrol/deploylicense/help.go

```diff
@@ -0,0 +1,11 @@
+package deploylicense
+
+const Description string = "Deploy a license from the specified bucket to an existing JPD. You may also deploy a number of licenses to an Artifactory 5.x HA cluster."
```

Comment:
Should `Artifactory 5.x HA` be `Artifactory HA`? These endpoints are for Mission Control v. 4.0 which doesn't support versions before v. 7.x. It seems better to not specify a version here.